crowdin.yml

```diff
@@ -1,9 +1,33 @@
+project_id: 527906
+api_token_env: CROWDIN_PERSONAL_TOKEN
```

Comment:
This is mainly used in conjunction with the vscode plugin. For details, please refer to: https://marketplace.visualstudio.com/items?itemName=Crowdin.vscode-crowdin